docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -25,8 +27,8 @@ Due to dependencies (for example ansible -> paramiko -> pynacl -> libffi):
     sudo apt update
     sudo apt install build-essential libssl-dev libffi-dev python-dev
 
-Common environment setup
-------------------------------
+Creating a common development environment
```

Comment:
The original heading meant "Environment set-up steps that are common across most/all platforms" (as opposed to steps that are specific to Ubuntu). The revised heading changes the meaning. Maybe this could be "Creating a development environment (platform-agnostic steps)"?